### ISSUE TITLE
How to find accuracy of a pretrained tflite model

### ISSUE THREAD

**Issue by suyash-narain** [tensorflow/tensorflow#60023]
### System information

-   **Have I written custom code (as opposed to using a stock example script
    provided in TensorFlow)**: No
-   **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- Ubuntu 20.04
-   **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue
    happens on a mobile device**: None
-   **TensorFlow installed from (source or binary)**: binary
-   **TensorFlow version (use command below)**: v2.11
-   **Python version**: 3.9

### Describe the problem
I have a pretrained tf mobilenetv2 model downloaded from tfhub which i converted to tflite using tflite interpreter converter. The original model was trained on imagenet dataset, but i want to find out the accuracy of the converted tflite model. How do I do that? As of now, the eval function is specific to models generated from modelmaker. What is the best way to find the model accuracy?

thanks

**Comment by pjpratik** [comment]
Hi @suyash-narain 

Tensorflow lite provides [TFLite Model Task Evaluation](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/tools/evaluation/tasks#tflite-model-task-evaluation) where we can perform the evaluation of image classification and object detection models.

For checking the imagenet accuracy, please refer to image classification evaluation based on ILSVRC 2012 task [here](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/tools/evaluation/tasks/imagenet_image_classification#image-classification-evaluation-based-on-ilsvrc-2012-task).

Please let us know if it helps and feel free to close the issue if it is resolved.

Thanks.

**Comment by suyash-narain** [comment]
thank you! it helps
I will close the issue now
